index.bs

```diff
@@ -0,0 +1,26 @@
+<pre class='metadata'>
+Title: RDF/JS: SHACL-UI specification
+Shortname: shacl-ui
+Level: 1
+Status: w3c/CG-DRAFT
+Group: RDF JavaScript Libraries Community Group
+Repository: rdfjs/shacl-ui
+URL: https://github.io/rdfjs/shacl-ui/
+Editor: Thomas Bergwinkl, https://www.bergnet.org/
```

Comment:
I think myself and @danielbeeke were also accepted as editors. Or would that be a different meta field?

Reply:
It's the same field. This is just an initial PR to be able to put more content with other PRs. Once it's merged, create a new PR, add another `Editor` line with the URL, Email, and text you would like to see with your name in the spec.